### PR TITLE
HC-10 Footer section

### PR DESCRIPTION
**Pull request by BogdanPavlyk**
- Updated footer.html
- Updated footer.css

### CODE REVIEW

File: src/css/sections/footer.css [hunk 1]
```diff
@@ -1,9 +1,18 @@
 .footer {
-  padding: 20px 0;
+  width: 100%;
+  background: #121417;
```
Comment (VOSolyanik) [inline src/css/sections/footer.css]: var(--charcoal-black-color)

File: src/css/sections/footer.css [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+  color: #ffffff;
```
Comment (VOSolyanik) [inline src/css/sections/footer.css]: var(--white-color)

File: src/partials/footer.html [hunk 3]
```diff
@@ -1,15 +1,76 @@
...
+          32a M. Bazhana ave
+        </a>
+      </div>
+      <ul class="anchor-links">
+        <li><a href="#about-us">About us</a></li>
+        <li><a href="#lessons">Our lessons</a></li>
+        <li><a href="#teachers">Teachers</a></li>
+        <li><a href="#reviews">Reviews</a></li>
+        <li><a href="#footer">Contact us</a></li>
+      </ul>
```
Comment (VOSolyanik) [inline src/partials/footer.html:34]: semanticaly - to be wrapped in `nav`

File: src/partials/footer.html [hunk 4]
```diff
@@ -1,15 +1,76 @@
...
+
+    <ul class="social-links">
```
Comment (VOSolyanik) [inline src/partials/footer.html]: replace with `<load src="partials/components/social-links.html" extra-class="inverse"/>`

File: src/partials/footer.html [hunk 5]
```diff
@@ -1,15 +1,76 @@
...
+      </li>
+    </ul>
+
+    <div class="footer-bottom">
+      <p>©2023 English Excellence. All rights reserved.</p>
+      <ul>
+        <li><a href="#">Privacy Policy</a></li>
+        <li><a href="#">Terms of Service</a></li>
+      </ul>
+      <p class="footer-desc">
```
Comment (VOSolyanik) [inline src/partials/footer.html]: remove, I believe it is left from template

File: src/partials/footer.html [hunk 6]
```diff
@@ -1,15 +1,76 @@
...
-        href="https://goit.global/ua/"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        GoIT team
+      <a class="nav-logo" href="./index.html" aria-label="Site logo">
+        english<span class="accent">excellence</span>.
       </a>
-    </p>
+      <div class="contacts">
```
Comment (VOSolyanik) [inline src/partials/footer.html]: semanticaly - replace div with address

**Review by VOSolyanik — CHANGES_REQUESTED** [review]

**Review by yuratouch — DISMISSED** [review]
Approved

**Review by VOSolyanik — APPROVED** [review]

**Pull request merged**